SAPex/Models/Validators/SandboxValidator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using FluentValidation;
+
+namespace SAPex.Models.Validators
+{
+    public class SandboxValidator : AbstractValidator<SandboxViewModel>
+    {
+        public SandboxValidator()
+        {
+            // RuleFor(sandbox => sandbox.Id).NotNull();
+            RuleFor(sandbox => sandbox.Name).NotEmpty().WithMessage("Sandbox name can t be empty");
+            RuleFor(sandbox => sandbox.Description).NotEmpty();
```

Comment:
All these rules should be used in different cases. They can not be used for one model. Please discuss with mentors.

Reply:
in progress